### PR TITLE
setup + select all user

### PR DESCRIPTION
**Pull request by koosoku**

### CODE REVIEW

File: routes/users_routes.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
 const app = require('express').Router()
 const usersController = require('../controllers/users_controller')
+usersController.setUp()
```
Comment (silverAndroid) [inline routes/users_routes.js]: shouldn't something like this go inside `index.js`

File: models/users_model.js [hunk 2]
```diff
@@ -1,42 +1,46 @@
...
-	], function (err) {
-	  if (err) {
-	    console.error('There was an error', err.message, err.stack);
-	  }
-	  console.log('Shutting down');
-	  client.shutdown();
-	});
+module.exports.setup = () => {
+    console.log('Setting up the database')
+    const id = cassandra.types.Uuid.random()
```
Comment (silverAndroid) [inline models/users_model.js]: unused variable

File: routes/users_routes.js (hunk 1 above)
Comment (koosoku) [reply routes/users_routes.js]: essentially the same thing, I also didn't want to expose the controller directly to the index.js

File: models/users_model.js (hunk 2 above)
Comment (koosoku) [reply models/users_model.js]: good catch

File: routes/users_routes.js [hunk 3]
```diff
@@ -1,11 +1,10 @@
...
 
 	app.get('/', usersController.findAllUsers)
```
Comment (BrandonDanis) [inline routes/users_routes.js]: indentation? 👎

Comment (koosoku) [reply routes/users_routes.js]: will update

**Pull request merged**